flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStMapState.java

```diff
@@ -131,6 +131,7 @@ public byte[] serializeKey(ContextKey<K, N> contextKey) throws IOException {
     public byte[] serializeValue(UV value) throws IOException {
         DataOutputSerializer outputView = valueSerializerView.get();
         outputView.clear();
+        outputView.writeBoolean(false);
```

Comment:
I am curious why you added this line? Maybe a comment would help - as `writeBoolean(false)` does not tell me much about what the intent is.

Reply:
This PR intents to align the sync state and async state format for serialization, for the convenient of state migration between sync and async state mode  ----- but this is still under planning